### ISSUE TITLE
Are there anyone Who Could Write Conf File for magento 

### ISSUE THREAD

**Issue by ksaltik** [h2o/h2o#1880]
Are there anyone Who Could Write Conf File for magento

**Comment by utrenkner** [comment]
I agree that many (potential) users could be helped with out-of-the-box config files. 

However, when I look at Magento's sample configurations for Apache and Nginx, they are *very* different in what they do. E.g. Apache is told to send out ``X-UA-Compatible`` headers, whereas this option is commented out in the sample Nginx configuration. The Nginx config foresees special treatment of the installer and update tools whereas there is nothing comparible in the sample .htaccess file. The [Apache config blocks various (sensitive) files](https://github.com/magento/magento2/blob/2.3-develop/.htaccess.sample#L179-L343), whereis Nginx is much more relaxed in what delivers.

Writing a basic Magento config for h2o should be rather simple. But first someone should write down what that config is supposed to do (taking the Apache and Nginx config files as basis).
